lang/en/texts/contribute-to-open-food-facts-by-playing.html

```diff
@@ -0,0 +1,74 @@
+<!-- no side column -->
+
+<h1>Play and contribute to Open Food Facts!</h1>
+
+<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteers developers.</h2>
+
+<div class="button_div unselectbuttondiv_front_en">
+	<a href="https://hunger.openfoodfacts.org/" class="button">Play now</button></a>
+<div>
+<p>We thank the many volunteers who have already <a href="https://en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
```

Comment:
```suggestion
<p>We thank the many volunteers who have already <a href="https://world-en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
```

Reply:
https://world-en.openfoodfacts.org/help-complete-products redirects me to https://world.openfoodfacts.org/help-complete-products